src/main/java/com/inquirer/controller/TestPageController.java

```diff
@@ -53,12 +58,11 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         request.setAttribute("questionTitle", questionTitle);
         request.setAttribute("answers", answers);
```

Comment:
You can use this interface to put necessary attributes 
http://docs.spring.io/spring-framework/docs/4.3.x/javadoc-api/org/springframework/ui/Model.html